gensim/models/rpmodel.py

```diff
@@ -5,41 +5,54 @@
 # Licensed under the GNU LGPL v2.1 - http://www.gnu.org/licenses/lgpl.html
 
 
-import logging
+"""
+Objects of this class allow building and maintaining a model for Random Projections
+(also known as Random Indexing). 
 
-import numpy as np
+For theoretical background on RP, see:   Kanerva et al.: "Random indexing of text samples for Latent Semantic Analysis."
 
-from gensim import interfaces, matutils, utils
+    The main methods are:
 
+    1. constructor, which creates the random projection matrix
+    2. the [] method, which transforms a simple count representation into the TfIdf
+       space.
 
-logger = logging.getLogger('gensim.models.rpmodel')
+Model persistency is achieved via its load/save methods.
 
 
-class RpModel(interfaces.TransformationABC):
-    """
-    Objects of this class allow building and maintaining a model for Random Projections
-    (also known as Random Indexing). For theoretical background on RP, see:
+Examples:
+---------
+>>> from gensim.models import rpmmodel
+>>> rp = RpModel(corpus)
+>>> print(rp[some_doc])
+>>> rp.save('/tmp/foo.rp_model')
+"""
 
-      Kanerva et al.: "Random indexing of text samples for Latent Semantic Analysis."
+import logging
 
-    The main methods are:
+import numpy as np
 
-    1. constructor, which creates the random projection matrix
-    2. the [] method, which transforms a simple count representation into the TfIdf
-       space.
+from gensim import interfaces, matutils, utils
 
-    >>> rp = RpModel(corpus)
-    >>> print(rp[some_doc])
-    >>> rp.save('/tmp/foo.rp_model')
 
-    Model persistency is achieved via its load/save methods.
-    """
+logger = logging.getLogger('gensim.models.rpmodel')
+
+
+class RpModel(interfaces.TransformationABC):
 
     def __init__(self, corpus, id2word=None, num_topics=300):
         """
         `id2word` is a mapping from word ids (integers) to words (strings). It is
         used to determine the vocabulary size, as well as for debugging and topic
         printing. If not set, it will be determined from the corpus.
+
+
+        Parameters
+        ----------
+        corpus : interfaces.CorpusABC
```

Comment:
please use 
```
:class:`~gensim.interfaces.CorpusABC`
```

when you reference a class from anywhere (gensim or something else)